lib/libs/webforms/CS12/v202401.ts

```diff
@@ -0,0 +1,483 @@
+import { DefaultFieldGroupProps, FormSchema, RHFSlotProps } from "shared-types";
+import { noLeadingTrailingWhitespace } from "shared-utils";
+
+const ageOptions = [
```

Comment:
Just a thought, we could consider generating this dynamically to make it more concise and easier to maintain, maybe something like this?
```
const ageOptions = Array.from({ length: 20 }, (_, i) => ({
  value: i.toString(),
  label: i.toString(),
}));
```

Reply:
I like this, but we'll need to update the other age forms as well to math if we do it